packages/toolpad-core/package.json

```diff
@@ -38,7 +38,7 @@
   },
   "homepage": "https://github.com/mui/toolpad#readme",
   "scripts": {
-    "clean": "rimraf build",
+    "clean": "rimraf --glob build \"*.tsbuildinfo\"",
```

Comment:
The two changes in this PR feel like they're at conflict with each other. Either we generate tsbuildinfo files in the build folder and then the old clean command would suffice. Or we generate them outside of the build folder and then we don't remove them during the copy script. But maybe there are reasons to keep this? I know there are differences in build between studio packages and core packages.